addons/kiam/3.x.0/kiam-1.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kiam
+  labels:
+    kubeaddons.mesosphere.io/name: kiam
+    kubeaddons.mesosphere.io/provides: iam
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "3.5.0-1"
+    appversion.kubeaddons.mesosphere.io/kiam: "3.5.0"
+    values.chart.helm.kubeaddons.mesosphere.io/kiam: "https://raw.githubusercontent.com/uswitch/kiam/master/helm/kiam/values.yaml"
+spec:
+  namespace: kube-system
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  cloudProvider:
+    - name: aws
+      enabled: true
```

Comment:
Do we want this enabled by default? cc @samba @jieyu

Reply:
if it is working as expected a 100% yes.
this is how we should sell it!
role assuming is more secure with kiam and can be restricted to namespace etc

Reply:
At this time IMO we should *not* have this in the base addons repo, as this is very specific to a single cloud provider. When we can support similar integrations on other cloud providers, I'd reconsider.